.planning/codebase/STRUCTURE.md

```
@@ -0,0 +1,126 @@
+# Structure
+
+## Top-Level Layout
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)